Header header
uint64 height
uint64 width
string name
